source/main/environment.js

```diff
@@ -41,7 +41,7 @@ const isDevelopment = checkIsDevelopment(NETWORK);
 const isWatchMode = process.env.IS_WATCH_MODE;
 const keepLocalClusterRunning = process.env.KEEP_LOCAL_CLUSTER_RUNNING;
 const API_VERSION = process.env.API_VERSION || 'dev';
-const NODE_VERSION = '1.30.1'; // TODO: pick up this value from process.env
+const NODE_VERSION = '1.32.1'; // TODO: pick up this value from process.env
```

Comment:
Why not 1.33.0?

Reply:
Because 1.33 needs to be first integrated in cardano-wallet, then needs to be tested on in Daedalus in all supported systems (win10/win11/mac-intel/mac-m1/linux) by doing a long regression test including tree kind of hardware wallets multipled by every operating system. Then we need a popup warning telling users to perform all urgent transactions before upgrading because the node need a full sync that needs hours. And so on. In other words we prioritize quality over fast release that is why it need time 😊. Thank you for understanding

Reply:
Thanks for explaining 🙏🏼